docs/pages/2019_IIT_Mandi_India/sections/first_taste.rst

```diff
@@ -52,23 +52,24 @@ Each piece of data in AiiDA gets a PK number (a "primary key") that identifies i
 The PK is printed to screen by the ``verdi data structure import`` command.
 **Mark down the PK for your structure and use it to replace the <PK> placeholders in what follows.**
 
-.. note::
+.. tip::
 
    You can view the structure either `online <http://crystallography.net/cod/9008565.html>`_
-   or use ``jmol 9008565.cif`` locally.
+   or by executing the ``jmol 9008565.cif`` command on the virtual machine.
```

Comment:
This may be very slow if run on the VM (just tried it).
A parenthesis of this may be added, with a suggestion to open it locally, if one has ``jmol`` installed.

Reply:
I've created a suggested change comment in response to your comment.